### ISSUE TITLE
[4.0] [backend-template] Edit page top menu 

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#25872]
### Steps to reproduce the issue

In the Joomla backend, go to edit an article, users (any edit page basically)

The top menu items become disabled as expected

### Expected result

A disabled identifier
(In Joomla 3, I believe the opacity of the text is changed)

### Actual result

They look exactly the same as they do when you can interact with them

![Screenshot_2019-08-15 Users Edit Profile - J4 - Administration](https://user-images.githubusercontent.com/2019801/63102256-b4ae2780-bf72-11e9-81e9-0ef483a54b2e.png)

**Comment by ghost** [comment]
Hover Button show like below "Multilingual Status", but i guess its about not to hover to see what is clickable (in J3 the Opacity is reduced).

![Bildschirmfoto 2019-08-16 um 07 39 53](https://user-images.githubusercontent.com/8235763/63145913-832c6f00-bff9-11e9-8a4a-e43dc8005fbc.png)

**Comment by infograf768** [comment]
We have to modify `/administrator/templates/atum/scss/blocks/_header.scss`
add after line 114 something like
`opacity: 0.3;`

```
        &.disabled,
        &.disabled:hover,
        &.disabled[aria-expanded="true"] {
          background-color: transparent;
          color: var(--atum-special-color);
          box-shadow: none;
          opacity: 0.3;
        }
```

and we will get:
<img width="572" alt="Screen Shot 2019-08-16 at 10 21 39" src="https://user-images.githubusercontent.com/869724/63153899-b5e16200-c00f-11e9-83ce-1223f17aa292.png">

**Comment by ghost** [comment]
Closed as having Pull Request #25906
